date_range/readme/CONFIGURE.rst

```diff
@@ -12,7 +12,7 @@ inherit from `date.range.search.mixin`:
 
 This will make a `Period` field show up in the search view:
 
-  .. figure:: https://raw.githubusercontent.com/OCA/server-tools/12.0/date_range/static/description/date_range_many2one_search_field.png
+  .. figure:: https://raw.githubusercontent.com/OCA/server-ux/15.0/date_range/static/description/date_range_many2one_search_field.png
```

Comment:
The correct way to put it and forget about this on each version is to put a relative path, and let the README generator to put the absolute one, but meanwhile, you see the correct image in local devel env.

```suggestion
  .. figure:: ../static/description/date_range_many2one_search_field.png
```

Reply:
@pedrobaeza you're right. I just checked the readme generator and relative paths are replaced by the right absolute path at generation.